src/layouts/Home.vue

```diff
@@ -144,7 +144,10 @@ $areas: main nav logo actions overlay survivors globe footer top;
           @apply text-h1;
         }
         display: inline-grid;
+        display: flex;
+        flex-direction: column;
```

Comment:
The `display:flex` here would be overriding the `inline-grid`

Changing it to flex is probably fine, but if that's what you wanna do, make sure you remove the older/dead style code though.